README.md

```diff
@@ -93,7 +93,7 @@ Make sure it is installed before running the next commands:
 
 ```
 $ pip3 install -r requirements.txt
-$ python3 setup.py install
+$ pip3 install . (-e for developer build)
```

Comment:
I think it will be more clear to describe developer installation in a different paragraph. For instance, developers need to install the dependencies of **requirements_tests.txt**.

Reply:
I'm sorry, but I don't get it:

* If you want to install from the repo, and you're a developer, instructions should be (AFAIU)

```
pip3 install -e .
```

No need to install `requirements.txt` in this case.

If you're not a developer, it would be better, I think to produce the package and then install it (but I'm not sure why you would like to install it from source if you're not a developer):

```
python3 setup.py sdist
pip3 install dist/package_name
```

Again, no need for installing requirements.

Or maybe I'm missing something?